backend/homework/9/src/main/java/com/example/springboot/Logging/Logg.java

```diff
@@ -0,0 +1,29 @@
+package com.example.springboot.Logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Profile;
+public class Logg{
+    Logg(){}
+    public static final Logger log = LoggerFactory.getLogger(Logg.class);
```

Comment:
The profiling in logs could also have been achieved by defining scope and levels in logger config file.